Sources/Amplitude/Amplitude.m

```diff
@@ -929,7 +930,7 @@ - (void)makeEventUploadPostRequest:(NSString *)url events:(NSString *)events num
     [postData appendData:[checksum dataUsingEncoding:NSUTF8StringEncoding]];
 
     [request setHTTPMethod:@"POST"];
-    [request setValue:@"application/x-www-form-urlencoded" forHTTPHeaderField:@"Content-Type"];
+    [request setValue:self.contentTypeHeader forHTTPHeaderField:@"Content-Type"];
```

Comment:
Should you set a default value if it's nil?

Reply:
Forgot to add that in init method, should be fixed now